modules/gitops-argo-rollouts-cli-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-cli-overview_{context}"]
+= Argo Rollouts CLI overview
+
+Argo Rollouts CLI is an optional plugin that allows you to handle and monitor Argo Rollouts resources directly through the command line interface instead of using {OCP} web console or the `oc` command. 
+
+The Argo Rollouts CLI plugin enables you to perform many actions, including the following actions:
```

Comment:
@maxwelldb, can we keep it as follows instead of your suggestion? 

The Argo Rollouts CLI plugin enables you to perform many actions, including the following:

Reply:
It's less redundant, but even my suggestions wasn't perfect with respect to ISG. Strictly speaking, we should never be using "the following" as an adjective.

Reply:
So @maxwelldb, shall I remove the adjective and just keep the sentence as follows:

 The Argo Rollouts CLI plugin enables you to perform many actions, including:

Reply:
Your original is correct per ISG. It uses "actions" redundantly, but it's the correct choice for translation. 

Other note, since I'm back in here:

⚠️ **enable** Use to describe the action of making an interface control or product function available to users. Avoid stating that inanimate objects grant abilities to people, as in “the product enables you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to …” or “with this product, users can…”.

So, all together, maybe:

```
With the Argo Rollouts CLI plugin, you can perform the following actions:
...

For a complete list of Argo Rollouts CLI plugin actions [commands?], see ...
```

I might even remove that list entirely if I'm in your shoes, as "manage and monitor" kind of gets at all of these things already, and the list is non-exhaustive, anyway. 

Ultimately, up to you! Peer review is advisory.

Reply:
Hi @maxwelldb - Thanks for the continued interest in improving the content further, I would like to stick to your next piece of suggestion for improving the lead-in sentence. However, there is no additional section that can provide information for a detailed list of Argo Rollouts CLI plugin actions so I am not including that line now.